src/main/js/main_html5.js

```diff
@@ -1521,7 +1521,7 @@ require("../../../html5-common/js/utils/environment.js");
     var dequeuePlay = _.bind(function() {
       if (playQueued) {
         playQueued = false;
-        executePlay(false);
```

Comment:
Do we still need the execute play function? Or is it used elsewhere?

Reply:
It is used for priming and does still need to remain separate.